src/JSInterop/Microsoft.JSInterop/test/DotNetObjectReferenceJsonConverterTest.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text.Json;
+using System.Threading.Tasks;
+using Xunit;
+using static Microsoft.JSInterop.TestJSRuntime;
+
+namespace Microsoft.JSInterop.Tests
+{
+    public class DotNetObjectReferenceJsonConverterTest
+    {
+        [Fact]
+        public Task Read_ReadsJson() => WithJSRuntime(_ =>
+        {
+            // Arrange
+            // Throw-away value
+            DotNetObjectRef.Create(new TestModel());
```

Comment:
What's the purpose of doing this?

Reply:
Making sure we're reviving the right object.

Reply:
Sorry I super don't get it 😆 this isn't used anywhere - what's the side-effect that's important?

Reply:
I can add a more detailed comment. Creating adds an id which is what we use to look up things. I want to verify we're restoring the right instance on deserialization.

Reply:
Still waiting for that more detailed comment.

Reply:
How does this do anything? - it seems like it validates some code we didn't write doesn't exist.

Reply:
It's validating that we're dereferencing the correct instance when we track more than one instance. Do you have suggestions about how we would validate this or what text would help you understand this test better?

Reply:
It would make a lot more sense if you compared the object ids to show that they are different then. A comment like `throw away value` should be replace with "we're doing this for the side effect of making sure that more than one value is being tracked"